pkg/services/publicdashboards/database/database.go

```diff
@@ -71,6 +70,17 @@ func (d *PublicDashboardStoreImpl) FindDashboard(ctx context.Context, dashboardU
 	return dashboard, err
 }
 
+func (d *PublicDashboardStoreImpl) Delete(ctx context.Context, userOrgId int64, dashboardUid string, uid string) error {
+	dashboard := &PublicDashboard{OrgId: userOrgId, DashboardUid: dashboardUid, Uid: uid}
+	return d.sqlStore.WithTransactionalDbSession(ctx, func(sess *db.Session) error {
+		_, err := sess.Delete(dashboard)
+		if err != nil {
+			return err
+		}
+		return nil
+	})
```

Comment:
Maybe we would like to distinguish a successful delete from a no data found attempt